docs/whats-new/toc.yml

```diff
@@ -2,8 +2,9 @@
   href: index.yml
   expanded: true
   items:
+    - name: 2020 - January
```

Comment:
this is not consistent with the 2019 node.
Shouldn't this be a 2020 node with a January child?

Reply:
Ah, never mind. Just saw your comment. Seems like an eng bug that should be fixed.

Reply:
I agree. But it won't be deployed in time before Feb becomes the "what's new" page.